runtime/doc/change.txt

```diff
@@ -540,29 +540,36 @@ with ".".  Vim does not recognize a comment (starting with '"') after the
 
 							*!*
 !{motion}{filter}	Filter {motion} text lines through the external
```

Comment:
brackets indicate optional args, in the docs.
```suggestion
!{motion}[filter]	Filter {motion} text lines through the external
```

Reply:
Thank you for this correction.